service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ManageOrdersMidEventControllerTest.java

```diff
@@ -383,11 +383,12 @@ void epoEndDateShouldReturnErrorWhenEndDateIsNotInRangeWithApprovalDate() {
         final LocalDateTime approvalDate = LocalDateTime.now().minusDays(5);
 
         CaseData caseData = buildCaseData().toBuilder().manageOrdersEventData(
-            buildRemoveToAccommodationEventData(approvalDate, approvalDate.plusDays(8).plusSeconds(1))).build();
+            buildRemoveToAccommodationEventData(approvalDate, approvalDate.plusDays(365).plusSeconds(1))).build();
 
         AboutToStartOrSubmitCallbackResponse response = postMidEvent(caseData, "order-details");
 
-        assertThat(response.getErrors()).containsOnly("Emergency protection orders cannot last longer than 8 days");
```

Comment:
just a minor thing: END_DATE_RANGE_MESSAGE is being used thrice, can we use this variable rather than string in all places?